modules/phase_field/src/kernels/CHParsed.C

```diff
@@ -54,13 +56,13 @@ CHParsed::computeGradDFDCons(PFFunctionType type)
   {
     case Residual:
       for (unsigned int i = 0; i <= _nvar; ++i)
-        res += (*_grad_vars[i])[_qp] * (*_second_derivatives[i])[_qp];
+        res += (*_grad_vars[i])[_qp]*(*_second_derivatives[i])[_qp];
       return res;
 
     case Jacobian:
-      res = _grad_phi[_j][_qp] * (*_second_derivatives[0])[_qp];
+      res = _grad_phi[_j][_qp]*(*_second_derivatives[0])[_qp];
       for (unsigned int i = 0; i <= _nvar; ++i)
-        res += _phi[_j][_qp] * (*_grad_vars[i])[_qp] * (*_third_derivatives[i])[_qp];
+        res += _phi[_j][_qp]*(*_grad_vars[i])[_qp]*(*_third_derivatives[i])[_qp];
```

Comment:
I have to touch these files in my big PR anyways, so unless @tonkmr is vehemently opposed to it I could restore them.